aten/src/ATen/cpu/vec256/vsx/vsx_helpers.h

```diff
@@ -0,0 +1,327 @@
+#pragma once
+#include <cstdint>
+#include <ATen/cpu/vec256/intrinsics.h>
+#define __inline_attrs  __attribute__((__always_inline__))
```

Comment:
Also here and at other places: Anything with leading double underscore is reserved and must not be used. Same for leading single underscore and capital letter.

Reply:
I see. I actually saw __ lowercase + __ usage and though the team ok with it. But I will better rename it to FORCE_INLINE

Reply:
What is usually done is that such macros are namespaces, e.g. `AT_FORCE_INLINE` (or maybe `ATEN_...`, check other macros for what they use)

Reply:
[__at_align32__](https://github.com/pytorch/pytorch/blob/8819bad86cd15c74dd076012c59e7b1f6081fda4/aten/src/ATen/cpu/vec256/vec256_base.h#L37)

Reply:
That is unfortunate and IMO should be changed. Other macros are e.g. `AT_ASSERTM`